test/unit/test_utils.js

```diff
@@ -61,6 +65,8 @@ function buildGetDocumentParams(filename, options) {
   params.url = isNodeJS
     ? TEST_PDFS_PATH + filename
     : new URL(TEST_PDFS_PATH + filename, window.location).href;
+  params.standardFontDataUrl = STANDARD_FONT_DATA_URL;
+  params.standardFontDataWorkerFetch = !isNodeJS;
```

Comment:
This line shouldn't be necessary, given the default value set in `src/display/api.js`, as far as I can tell.